README.md

```diff
@@ -57,21 +57,13 @@ The package can be installed via
 pip install fairlearn
 ```
 
+and optionally with extensions such as `pip install fairlearn[widget]` to get the fairlearn dashboard.
```

Comment:
so its required to specify that we want to include this?

Reply:
Yeah in the current version that's true, but that won't work for us because it'll break users like you said :-(

Reply:
this is not relevant anymore since fairlearn as of now will include the widget